src/components/Card.tsx

```diff
@@ -26,8 +32,9 @@ const Card: React.FC<IProps> = ({
     borderColor="lightBorder"
     p={6}
     className={className}
+    style={{ justifyContent: "space-between" }}
```

Comment:
`Flex` is a Chakra component. The correct way to express this would be:
```suggestion
    justify="space-between"
```
See more on https://chakra-ui.com/docs/components/flex/usage